belt_nlp/bert.py

```diff
@@ -33,6 +33,7 @@ def __init__(
         batch_size: int,
         learning_rate: float,
         epochs: int,
+        accumulation_steps: Optional[int] = 1,
```

Comment:
Do we need Optional here and in other places?

Reply:
I think so.  We just want this to default to 1 in case it is not passed by the user.  Accumulation steps = 1 is equivalent to no gradient accumulation.

Reply:
Optional typing hint allows for the `None` value. I suggest leaving just `int` type, The same way it is type hinted in lightning: https://github.com/Lightning-AI/pytorch-lightning/blob/7cc79fe7bad6b6faac0a7889ad81c4050bc32c86/src/lightning/pytorch/trainer/trainer.py#L121

Reply:
Ah, makes sense - thank you!